### ISSUE TITLE
Lensfun compile instructions

### ISSUE THREAD

**Issue by bronger** [lensfun/lensfun#791]
I followed [these](http://lensfun.sourceforge.net/development/) instructions on Ubuntu 17.04. Works great until you try to compile Darktable.

Darktable builds but the lens module is not there. Darktable 2.2.3 complains in the terminal about not finding the lensfun module. 2.2.4 is even more vague, it only complains if you try to open a picture which (on a previous version of Darktable) uses the lensfun module.

Long story short it is an easy fix. Run ldconfig after installing darktable. Then build Darktable. This might be obvious to the more seasoned linux user but not to me (it cost me a few hours to figure it out). You might want to include it at the end of the instructions to install Lensfun from source.

Reported by: computerfreaked
